### ISSUE TITLE
SmartCity-Accessibility #371 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#205]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
Started by user Alexander Kaplan
Building in workspace /var/jenkins_home/workspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6
 > git rev-list 01639875de8008bb8f6ad5a848a7f0ed3ba95931 # timeout=10
Copied 6 artifacts from "SmartCity-Accessibility-Deps" build number 11
[SmartCity-Accessibility] $ mvn clean -Dram=2048 jacoco:prepare-agent test jacoco:report
FATAL: command execution failed
java.io.IOException: Cannot run program "mvn" (in directory "/var/jenkins_home/workspace/SmartCity-Accessibility"): error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at hudson.Proc$LocalProc.<init>(Proc.java:245)
	at hudson.Proc$LocalProc.<init>(Proc.java:214)
	at hudson.Launcher$LocalLauncher.launch(Launcher.java:846)
	at hudson.Launcher$ProcStarter.start(Launcher.java:384)
	at hudson.Launcher$ProcStarter.join(Launcher.java:395)
	at hudson.tasks.Maven.perform(Maven.java:367)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:534)
	at hudson.model.Run.execute(Run.java:1728)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:404)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 15 more
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [GitHub] fcbedc3fd5e2e3ab6060cfb198d1f7bcc8cb1569 - Update README.md

- [Yossi Gil] bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 - Spartanize

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/371/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
Started by user Alexander Kaplan
Building in workspace /var/jenkins_home/workspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6
 > git rev-list bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 # timeout=10
Copied 6 artifacts from "SmartCity-Accessibility-Deps" build number 11
[SmartCity-Accessibility] $ mvn clean -Dram=2048 jacoco:prepare-agent test jacoco:report
FATAL: command execution failed
java.io.IOException: Cannot run program "mvn" (in directory "/var/jenkins_home/workspace/SmartCity-Accessibility"): error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at hudson.Proc$LocalProc.<init>(Proc.java:245)
	at hudson.Proc$LocalProc.<init>(Proc.java:214)
	at hudson.Launcher$LocalLauncher.launch(Launcher.java:846)
	at hudson.Launcher$ProcStarter.start(Launcher.java:384)
	at hudson.Launcher$ProcStarter.join(Launcher.java:395)
	at hudson.tasks.Maven.perform(Maven.java:367)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:534)
	at hudson.model.Run.execute(Run.java:1728)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:404)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 15 more
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [GitHub] fcbedc3fd5e2e3ab6060cfb198d1f7bcc8cb1569 - Update README.md

- [Yossi Gil] bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 - Spartanize



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/372/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 4.25 MB...]
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.008 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 0.007 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: IPC process exited. Exit code: 127
	at com.teamdev.jxmaps.internal.internal.ipc.p.run(SourceFile:185)
	at java.lang.Thread.run(Thread.java:745)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 2, Skipped: 1, Time elapsed: 0.018 sec <<< FAILURE! - in smartcity.accessibility.search.SearchTest
test2(smartcity.accessibility.search.SearchTest)  Time elapsed: 0.006 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: IPC process exited. Exit code: 127
	at com.teamdev.jxmaps.internal.internal.ipc.p.run(SourceFile:185)
	at java.lang.Thread.run(Thread.java:745)

test3(smartcity.accessibility.search.SearchTest)  Time elapsed: 0.009 sec  <<< ERROR!
com.teamdev.jxmaps.internal.internal.ipc.v: IPC process exited. Exit code: 127
	at com.teamdev.jxmaps.internal.internal.ipc.p.run(SourceFile:185)
	at java.lang.Thread.run(Thread.java:745)


Results :

Tests in error: 
  NavigationTest.avoidTwoSegement » v IPC process exited. Exit code: 127
  NavigationTest.displayMap:72->Object.wait:502->Object.wait:-2 » TestTimedOut t...
  NavigationTest.avoidOneSegement » v IPC process exited. Exit code: 127
  NearbyPlacesTest.nearByPlacesTest » v IPC process exited. Exit code: 127
  SearchTest.test2 » v IPC process exited. Exit code: 127
  SearchTest.test3 » v IPC process exited. Exit code: 127

Tests run: 58, Failures: 0, Errors: 6, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:53 h
[INFO] Finished at: 2017-03-20T17:34:40+00:00
[INFO] Final Memory: 28M/376M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/jenkins_home/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [GitHub] fcbedc3fd5e2e3ab6060cfb198d1f7bcc8cb1569 - Update README.md

- [Yossi Gil] bafc12582ed5901e3009a8ae8f75a5efa7fa8ac6 - Spartanize



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility/373/)

**Comment by yaelAmitay** [comment]
Does anyone know why it shows this error?
from a small online search i'v made i see it's relate to JxBrowser but I can't see why...

**Comment by kapalex** [comment]
I'm currently reactivating the jenkins server so you can ignore this for now
